apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -225,3 +253,78 @@ function PopoverEndDate() {
     </p>
   `.toString();
 }
+
+function CreateCourseInstanceModal({
+  csrfToken,
+  initialStartDateFormatted,
+  initialEndDateFormatted,
+}: {
+  csrfToken: string;
+  initialStartDateFormatted?: string;
+  initialEndDateFormatted?: string;
+}) {
+  return Modal({
+    id: 'createCourseInstanceModal',
+    title: 'Create Course Instance',
+    formMethod: 'POST',
+    body: html`
+      <div class="form-group">
+        <label for="short_name">CIID (Course Instance Identifier)</label>
+        <input
+          type="text"
+          class="form-control"
+          id="short_name"
+          name="short_name"
+          required
+          pattern="[\\-A-Za-z0-9_\\/]+"
+          title="Please enter a valid format: only letters, numbers, hyphens, underscores, backslashes, and forward slashes."
+        />
+        <small class="form-text text-muted"
+          >The recommended format is <code>Fa19</code> or <code>Fall2019</code>. Use only letters,
+          numbers, dashes, and underscores, with no spaces.
+        </small>
+      </div>
+      <div class="form-group">
+        <label for="long_name">Long Name</label>
+        <input type="text" class="form-control" id="long_name" name="long_name" required />
+        <small class="form-text text-muted">
+          This is the full name of the course instance, such as "Fall 2019" or "Spring 2020".
+        </small>
+      </div>
+      <div class="form-group">
+        <label for="start_access_date">Start Access Date</label>
+        <input
+          class="form-control date-picker"
+          type="datetime-local"
+          id="start_access_date"
+          name="start_access_date"
+          value="${initialStartDateFormatted}"
+          max="${initialEndDateFormatted}"
+        />
```

Comment:
In other contexts, we like to explicitly show the time zone that's being used, e.g.:

<img width="390" alt="Screenshot 2024-12-13 at 14 16 08" src="https://github.com/user-attachments/assets/45900090-e0f3-426b-92b9-e394637f6da7" />

That's implemented here:

https://github.com/PrairieLearn/PrairieLearn/blob/81a771b6e6d9b7cce6aa80851efa1c4d9da4ce59/apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.tsx#L595

Might be nice to do the same here as well!

Reply:
I added this:

![image](https://github.com/user-attachments/assets/92e998b9-a3a1-49c4-9d75-c9b935dee17c)